### PR TITLE
Fix annotations in puppetdb-pvc and puppetserver-pvc

### PR DESCRIPTION
**Pull request by akiryushin**

### CODE REVIEW

**Comment by davidphay** [comment]
Thanks for the PR! can you please sign the CLA & bump the `Chart.yaml` version, update the `changelog.md` & You might want to include your name in the [Credits](https://github.com/puppetlabs/puppetserver-helm-chart#credits)?

**Review by Xtigyro — DISMISSED** [review]
Thanks for the contribution, @akiryushin! 🥇

**Comment by akiryushin** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version `v0.2.4` is auto-generated.
 
+## [v7.1.1](https://github.com/puppetlabs/puppetserver-helm-chart/tree/v7.0.0) (2023-01-xx)
```
Comment (davidphay) [inline CHANGELOG.md]:
```suggestion
## [v7.1.1](https://github.com/puppetlabs/puppetserver-helm-chart/tree/v7.2.2) (2023-01-18)
```

Please update the date here

File: templates/puppetserver-pvc.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ metadata:
     "helm.sh/hook-delete-policy": "before-hook-creation"
     {{- end }}
     {{- if .Values.storage.annotations }}
-    {{ toYaml .Values.storage.annotations }}
+    {{ toYaml .Values.storage.annotations | indent 4 }}
```
Comment (davidphay) [inline templates/puppetserver-pvc.yaml]:
```suggestion
    {{ toYaml .Values.storage.annotations | nindent 4 }}
```

**Comment by davidphay** [comment]
I add 2 suggestions, after that it will be ok for me.

**Comment by davidphay** [comment]
@akiryushin Hello, you have to rebase your branch, many commit has been added.

**Comment by akiryushin** [comment]
@davidphay done

File: templates/puppetserver-pvc.yaml (hunk 2 above)
Comment (davidphay) [reply templates/puppetserver-pvc.yaml]: @akiryushin Here it's not fixed

**Comment by davidphay** [comment]
@akiryushin in `templates/puppetserver-pvc.yaml` there still an issue.

**Comment by akiryushin** [comment]
@davidphay sorry, didn't see. Check it

**Review by davidphay — APPROVED** [review]
Thanks @akiryushin !

File: CHANGELOG.md (hunk 1 above)
Comment (akiryushin) [reply CHANGELOG.md]: late?

**Pull request merged**